examples/quantize/quantize.cpp

```diff
@@ -224,13 +246,119 @@ static ggml_type parse_ggml_type(const char * arg) {
     for (int j = 0; j < GGML_TYPE_COUNT; ++j) {
         auto type = ggml_type(j);
         const auto * name = ggml_type_name(type);
-        if (name && strcmp(arg, name) == 0) {
+        if (name && strcasecmp(arg, name) == 0) {
             result = type; break;
         }
     }
     return result;
 }
 
+static bool parse_kv_override(const char * data, std::vector<llama_model_kv_override> & overrides) {
```

Comment:
This function was moved to `common.cpp`, this definition should be removed from here.